plexapi/library.py

```diff
@@ -543,6 +543,53 @@ def _cleanSearchSort(self, sort):
             raise BadRequest('Unknown sort dir: %s' % sdir)
         return '%s:%s' % (lookup[scol], sdir)
 
+    def sync(self, client, policy, media_settings, title=None, sort=None, libtype=None, **kwargs):
+        """ Add current library section as sync item for specified device.
+            See description of :func:`plexapi.library.LibraySection.search()` for details about filtering / sorting.
+
```

Comment:
Example is good but can you add docs string for args?

Reply:
Yep, sure, I'm working on tests now, and fixing all the docs :)